calculator_test.go

```diff
@@ -1,24 +1,136 @@
-package calculator_test
+package calculator
```

Comment:
I would keep this in package `calculator_test`, so that we're forced to import and use our own public API.